docs/en/writing-running-appium/ios-xctest-source.md

```diff
@@ -0,0 +1,22 @@
+## Getting Page Source For iOS With WebDriverAgent/XCTest Backend
+
+WebDriverAgent provides a possibility to represent page source in different formats. Usually it is enough to call _getPageSource_ driver method, which is the standard one and is available for all the client libraries. This method returns the hierarchical representation of the current UI source represented as XML document. However, sometimes it is convenient to get page representation in JSON format or some other format, which is supported by WDA/XCTest. Also, the actual performance of different page representation providers is different and the default method is not always necessarily the most performant one.
```

Comment:
This makes it seem like "xml", "json" and some other format(s) are available, but the `format` description below says only the two are there.

Reply:
yes, I want to add at least one more and there is a PR for that, however I don't know when it is going to be merged